internal/api/relationships.go

```diff
@@ -124,3 +124,54 @@ func (r *Router) relationshipsList(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, out)
 }
+
+func (r *Router) relationshipDelete(c echo.Context) error {
+	resourceIDStr := c.Param("id")
+
+	ctx, span := tracer.Start(c.Request().Context(), "api.relationshipsDelete", trace.WithAttributes(attribute.String("id", resourceIDStr)))
+	defer span.End()
+
+	resourceID, err := gidx.Parse(resourceIDStr)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest, "error parsing resource ID").SetInternal(err)
+	}
+
+	var reqBody deleteRelationshipRequest
```

Comment:
Why not use query params for this instead of a DELETE body?

Reply:
using a query parameter seems odd for a delete request. however having everything in the url path is pretty standard. I could change it to be `DELETE /resources/:id/relationships/:relation/:related_id`. Thoughts about that?

Reply:
i think this is fine for now 👍